app/services/SaveMessageService.scala

```diff
@@ -47,7 +47,10 @@ class SaveMessageService @Inject()(arrivalMovementRepository: ArrivalMovementRep
               .addResponseMessage(messageSender.arrivalId, message, arrivalStatus)
               .map {
                 case Success(_) => SubmissionSuccess
-                case Failure(_) => SubmissionFailureInternal
+                case Failure(e) => {
+
+                  SubmissionFailureInternal
+                }
```

Comment:
Did you want to propagate this back for logging?

Reply:
The logging for failures is handled at controller level. Don't want to change this here because it would mean considering the other failure scenarios (which let the controller handle them)